i18n/en_AU.json

```diff
@@ -913,6 +914,9 @@
   "admin.false": "false",
   "admin.featureDiscovery.WarningDescription": "Your Licence is being updated to give you full access to all the Enterprise Features. This page will automatically refresh once the licence update is complete. Please wait ",
   "admin.featureDiscovery.WarningTitle": "Your trial has started and updates are being made to your licence.",
+  "admin.feature_discovery.trial-request.accept-terms": "By clicking **Start trial**, I agree to the [Mattermost Software Evaluation Agreement](!https://mattermost.com/software-evaluation-agreement/), [Privacy Policy](!https://mattermost.com/privacy-policy/), and receiving product emails.",
```

Comment:
having a `!` within the link seems to break the desktop app, not sure if this is a desktop problem or a translation problem. 
Thoughts @devinbinnie @cwarnermm ?

Reply:
@ctlaltdieliet - Would this be a case where we'd want to remove the URL and replace it with a placeholder? Is the ``!`` symbol something that gets added as part of the placeholder? 

Related note: Searching for ``https:``, I see a few remaining URLs with in ``i18n/en.json``. Is this intended? Are these strings properly formatted?

Reply:
Yes it is this one!
I've seen the ! at other urls as well but it's not valid Markdown as for as I can find in the documentation. Maybe that is the reason that it breaks?
There are only 3 )[!http and 3 )[http strings in the en.json file. So almost all are gone. Some are really hardto replace by a variable as we would need to change the component they are used in.

Reply:
The `!` is a hack someone added to mean "open this link in another tab" when used in a link in a `FormattedMarkdownMessage`. It's not needed any more though since we've changed it so that all external links in `FormattedMarkdownMessage`s in the web app are automatically opened in a new tab. It shouldn't cause an error though as long as we aren't mixing rich formatting and Markdown in the same string.

I'll see why this one seems to be failing since I don't think there should be any issue even if the link is still prefixed by an exclamation mark

Reply:
@Willyfrog What error are you seeing? This link opens for me just fine.

We shouldn't be adding more links like this to the app though. I'll file a ticket to get this fixed, mark the `FormattedMarkdownMessage` component as deprecated (it shouldn't be needed any more with rich text formatting), and bring this up in the dev meeting so that we stop adding them

Reply:
I pasted such a link in a post and ended up in a channel not found screen
![Uploading CleanShot 2022-08-24 at 10.58.00.gif…]()

Reply:
ticket created to fix in the desktop: https://mattermost.atlassian.net/browse/MM-46580

Reply:
Another ticket filed to fix this in the component. Looks like the actual problem is that we're mixing Markdown and rich text formatting in this string

https://mattermost.atlassian.net/browse/MM-46609